src/Core/Animation/DualQuaternionSkinning.cpp

```diff
@@ -79,9 +79,10 @@ void computeDQ_naive( const Pose& pose, const WeightMatrix& weight, DQList& DQ )
     poseDQ.reserve( pose.size() );
 
     // 1. Convert all transforms to DQ
+#pragma omp parallel for
     for ( int j = 0; j < weight.cols(); ++j )
     {
-        poseDQ.push_back( DualQuaternion( pose[j] ) );
+        poseDQ[j] = DualQuaternion( pose[j] );
```

Comment:
Shouldn't the call to `reserve` be replaced by `resize` ?
You are accessing objects that are not yet allocated.